### PR TITLE
Update state channels' examples

### PR DESCRIPTION
**Pull request by velzevur**
This is the output of the aeternity/aeternity#2235

Files in `node/api/examples/channels/*/` are generated, so please take a look at the [new README.md](https://github.com/aeternity/protocol/blob/3348394432ed56fe1dc25499687847eab305d6cc/node/api/examples/channels/README.md)

### CODE REVIEW

**Comment by velzevur** [comment]
@lucafavatella Thank you for the fixes of the dead links

File: node/api/examples/channels/legacy/continuous/init_per_group.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+  "channel_id": "ch_2gouzi4HtoExEMDucQ96sT4da6wg73cvJuD7Su3Uwqaaq3bFtT",
+  "payload": {
+    "state": "tx_+QENCwH4hLhAmZbbVf4/NfmhVbnLEUzxP1zb/m/r1mHsBzQRIB5Ku2bGrKb7o6MvAcjX6PKSrdt4hsciL6H0xFnDRp5V3cJLAbhAoI+fvXfrhJ4kjSz+lgij4CS9ANvHmsDrvbkJeODGA6PHCNxnors/VotUYUZeLuM2F8mdSBBbFFTBSjK/yZQjBriD+IEyAaEBDB0foygfU35mpXAMOB5T3y5Uz+HxiwugHh8Qy/jI3nqGP6olImAAoQGlZyRxc6JXyoCZ1k5JANokwX6uj8/EZxGEGyc8x2MXK4YkYTnKgAACCgCGEjCc5UAAwKAVUCLa13l4tTTVS6xTUZMu5qobq+gq1dyIaegDWACVUga6kKf1"
+  },
+  "version": 1
+}
+```
+
+#### responder: (2019-03-13 11:13:48.972)
+> Funding has been confirmed locally on-chain
```
Comment (cytadela8) [inline node/api/examples/channels/legacy/continuous/init_per_group.md]: This comments shouldn't probably all be at the end. I think some write caching happened probably. (output wasn't flushed correctly and all messages were printed at the end.) This also applies to other transcripts, but weirdly to not all of them.

Comment (velzevur) [reply node/api/examples/channels/legacy/continuous/init_per_group.md]: It was a race condition, addressed in [80293cd](https://github.com/aeternity/aeternity/pull/2235/commits/80293cd7f62052d6c5ef4ea7de5eee89c5c23f00)

File: node/api/examples/channels/README.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+In the section, right after the participant, there is an arror indicating the
+direction of the message:
+
+* `<--- node` means that a message had been received from the node
+* `---> node` means that a message had been sent to the node
+
+##### Continuous example
+
+There are two types of examples: that show a full open-close scenario or those
+that are built on top of the previous ones'state. The latter are in `./*continuous/`
```
Comment (uwiger) [inline node/api/examples/channels/README.md]: `./*/continuous/`?

Comment (velzevur) [reply node/api/examples/channels/README.md]: Addressed

**Pull request merged**